### PR TITLE
Fix my deposits

### PR DESCRIPTION
**Pull request by Luigy-Lemon**
Similar change to My Deposits that has been already pushed for My Borrows

### CODE REVIEW

File: src/views/Borrow/index.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ export const Borrow: React.FC<{}> = () => {
   const [activeValue, setActiveValue] =
     React.useState<"All" | "Stable Coins">("All");
   const balances = useUserVariableDebtTokenBalancesDaiWei();
+  console.log(balances);
```
Comment (befitsandpiper) [inline src/views/Borrow/index.tsx]: remove console

**Review by befitsandpiper — APPROVED** [review]
nit: please remove console log then you can merge the PR.

File: src/views/Borrow/MyBorrows.tsx [hunk 2]
```diff
@@ -25,8 +25,20 @@ const AssetBalanceDisplay: React.FC<{ asset: BorrowAsset }> = ({ asset }) => {
...
+              placement="top-start"
+            >
+              <Text fontSize="smaller" onHover>
+                {bigNumberToString(asset.balance, 2)} x $
+                {bigNumberToString(asset.daiWeiPricePer, 2)}
+              </Text>
+            </Tooltip>
+            <Text fontWeight="bold">
               $ {bigNumberToString(asset.daiWeiPriceTotal)}
             </Text>
```
Comment (zedkai) [inline src/views/Borrow/MyBorrows.tsx]: Looks like this construct is repeated twice in this PR and at least once in https://github.com/Agave-DAO/agave-frontend/pull/52/files#diff-6d276eb2737964ef22c916f27e5c3f46e79f6ce2dba7d4ce39ddc5e01e16ff15R28-R41

As this is a derived commit from the PR #52, please revise *this* PR such that it turns that block from 52 into its own component that takes (`balance`, `daiWeiPricePer`, and `daiWeiPriceTotal`) *or* (`asset`) as parameters, put it in [`components`](https://github.com/Agave-DAO/agave-frontend/tree/dev/src/components), and use it in these two places to share the formatting and behaviours exactly.

File: src/views/Deposit/MyDeposits.tsx [hunk 3]
```diff
@@ -25,10 +25,21 @@ const AssetBalanceDisplay: React.FC<{ asset: DepositAsset }> = ({ asset }) => {
...
+              bg="secondary.900"
+              placement="top-start"
+            >
+              <Text fontSize="smaller" onHover>
+                {bigNumberToString(asset.balance, 2)} x $
+                {bigNumberToString(asset.daiWeiPricePer, 2)}
+              </Text>
+            </Tooltip>
+            <Text fontWeight="bold">
+              $ {bigNumberToString(asset.daiWeiPriceTotal)}
```
Comment (zedkai) [inline src/views/Deposit/MyDeposits.tsx]: Use the component created in response to my other comment.

**Review by zedkai — CHANGES_REQUESTED** [review]
Unnecessary code duplication.

**Review by zedkai — APPROVED** [review]
Nice deduplication.

File: src/views/Deposit/MyDeposits.tsx [hunk 4]
```diff
@@ -2,47 +2,14 @@ import React from "react";
...
-};
 
 const Deposits: React.FC<{ assets: DepositAsset[] }> = ({ assets }) => {
   const total = React.useMemo(() => {
     return assets.reduce(
-      (memo, next) =>
-        memo +
-        (Number( bigNumberToString(next.daiWeiPriceTotal)) ),
+      (memo, next) => memo + Number(bigNumberToString(next.daiWeiPriceTotal)),
       0
```
Comment (zedkai) [inline src/views/Deposit/MyDeposits.tsx]: Wait a minute- why are you using Number here? This should stay BigNumber until it becomes display-output-truncated.
If you're turning it into a display-value, do so *after* adding everything.

`reduce((memo, next) => next.daiWeiPriceTotal !== undefined ? memo.add(next.daiWeiPriceTotal) : memo, constants.Zero)`

**Review by zedkai — CHANGES_REQUESTED** [review]
Missed this one since it was from a prior PR.

**Review by zedkai — APPROVED** [review]

**Pull request merged**